Week3/homework/step3.js

```diff
@@ -2,7 +2,12 @@
 
 function createBase(base) {
   // Replace this comment and the next line with your code
-  console.log(base);
+  // console.log(base);
+  const num1 = base;
+  // return function name(x) {
+  //   return x + num1;
+  // };
+  return x => x + num1;
```

Comment:
You need not have created a new variable `num1` here. I think you can use `base` instead without a problem.

Reply:
yes, i don't need `num1`.